docs/user-guide/api-mediation/configuration-saf-resource-checking.md

```diff
@@ -2,68 +2,74 @@
+
+- **dummy**  
+dummy is the lowest priority provider. This is the dummy implementation and is defined in a file.
 
 :::note
-Verification of the SAF resource uses the first available provider based on the specified priority. The default configuration resolves to the native provider. 
+Verification of the SAF resource uses the first available provider based on the specified priority. The default configuration resolves to the **native** provider. 
 :::
 
-You can select a specific provider by specifying the `components.gateway.apiml.security.authorization.provider` key in the `zowe.yaml` file. Use the parameter value to
+Select a specific provider by specifying the `components.gateway.apiml.security.authorization.provider` key in the `zowe.yaml` file. Use the parameter value to
 strictly define a provider. If verification is disabled, select the `endpoint` option. 
 
 1. Open the file `zowe.yaml`.
 2. Find or add the property `components.gateway.apiml.security.authorization.provider` and set desired value.
 3. Restart Zowe.
 
 **Examples:**
-```
-components.gateway.apiml.security.authorization.endpoint.url: endpoint
-```
 
-To configure the `endpoint` provider, add the following additional property:
-`components.gateway.apiml.security.authorization.endpoint.enabled: true`
+* **Native**
+    `components.gateway.apiml.security.authorization.provider: native`
+    If you leave the property empty, this is used as the default value, even if you enable set `components.gateway.apiml.security.authorization.endpoint.enabled` to `true` (starting from version 3.1).
+* **Endpoint**
+    `components.gateway.apiml.security.authorization.provider: endpoint`
+* **Dummy**
+    `components.gateway.apiml.security.authorization.provider: dummy`
 
-`components.gateway.apiml.security.authorization.provider: native`
-
-`components.gateway.apiml.security.authorization.provider: dummy`
-
-
-To use the endpoint provider, customize the URL corresponding to the SAF resource authorization. By default, the ZSS API is configured and used. 
+To use the endpoint provider, you also need to enable the endpoint property and customize the URL corresponding to the SAF resource authorization. By default, the ZSS API is configured and used.
 
 1. Open the file `zowe.yaml`.
-2. Find or add the property `components.gateway.apiml.security.authorization.endpoint.url` and set desired value.
-   The default value for ZSS API is `https://${ZWE_haInstance_hostname}:${GATEWAY_PORT}/zss/api/v1/saf-auth`
+2. Find or add the folowing properties:
+   - `components.gateway.apiml.security.authorization.provider: endpoint` 
+   - `components.gateway.apiml.security.authorization.endpoint.enabled: true`
+   - `components.gateway.apiml.security.authorization.endpoint.url: <endpoint_url>`
+  
+   When using ZSS, the default value the property
+   `components.gateway.apiml.security.authorization.endpoint.url` is `https://${ZWE_haInstance_hostname}:${GATEWAY_PORT}/zss/api/v1/saf-auth`
```

Comment:
Please define the variables that require customization in the following format:

* ZWE_haInstance_hostname
Specifies....

* GATEWAY_PORT
Specifies ...